ui/src/keyboardMappings.ts

```diff
@@ -131,23 +164,23 @@ export const keyDisplayMap: Record<string, string> = {
   AltMetaEscape: "Alt + Meta + Escape",
   CtrlAltBackspace: "Ctrl + Alt + Backspace",
   Escape: "esc",
-  Tab: "tab",
-  Backspace: "backspace",
-  "(Backspace)": "backspace",
+  Tab: "tab ⇥",
```

Comment:
Added the cool unicode characters for the keys, so the keyboard looks nicer now.

Reply:
These were not pulled into #725 either, will revisit when finishing the localization of VirtualKeyboard